### DISCUSSION TITLE
AlDente 1.20

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#863] [General]
* Official launch of the Automatic Discharge feature! Read more about it on our blog: <a href = “https://apphousekitchen.com/feature-explanation-automatic-discharge/“>https://apphousekitchen.com/feature-explanation-automatic-discharge/</a>
* New beta feature “Control MagSafe LED” which allows you to control the MagSafe charging LED for all MagSafe 2 and MagSafe 3 MacBooks which are supported by AlDente Pro. Read more about it on our blog: <a href = “https://apphousekitchen.com/feature-explanation-control-magsafe-led/“>https://apphousekitchen.com/feature-explanation-control-magsafe-led/</a>
* Certain UI elements in settings can now be collapsed for better usability
* Reduced the size of the menu bar item if only the percentage is shown
* “Top Up” now overrides “Disable Charging when Sleeping”
* New warning and information messages to improve usability
* Changed some icons that caused the app to crash on some devices with older versions of macOS
* Added Portuguese language support
* Updated many language translations
* Many other improvements and bugfixes

Happy new year from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente-Charge-Limiter/releases/tag/1.20'>AlDente 1.20</a>.</em>

**Comment by StormForest** [comment]
To the operator,

AlDante Program Free Version 1.20 Even if you press the discharge button, the icon remains in the shape of an electric cord, and the battery is decreasing.

MacBook MagSafe charging light turned orange. There seems to be an error, but please provide a corrected version as soon as possible.

OS used is MacOS Ventura 13.2.

**Reply by MatthiasKerbl** [reply]
Hi @StormForest,

The icon of AlDente Free is static and does not change. Furthermore, when you activate the Discharge feature, your battery level will decrease and discharge to the set level in AlDente. You can read more about the Discharge feature on our blog here: https://apphousekitchen.com/feature-explanation-discharge/

When your MacBook is discharging, the MagSafe LED will be orange.

Cheers,
Matthias
